CONTRIBUTING.md

```diff
@@ -92,9 +92,37 @@ provide any backward compatible guarantees.
 
 #### Backporting
 
-We only accept backports of critical bugs, security issues, or bugs without easy
-workarounds, any backport MUST not be breaking for either API or behavioral
-changes. We generally do not accept PRs against older release branches.
+We generally do not accept PRs directly against release branches, while we might
+accept backports of fixes/changes already merged into the main branch.
+
+We generally allow backports of following changes to all supported branches:
+
+- Critical bugs fixes, security issue fixes, or fixes for bugs without easy
+workarounds.
+- Dependency bumps for CVE (usually limited to CVE resolution; backports of
+non-CVE related version bumps are considered exceptions to be evaluated case by
+case)
+- Changes required to support new Kubernetes versions, when possible.
```

Comment:
Maybe specify patch versions of Kubernetes, we don't support backporting new minors?

Reply:
good point